conversions/eastern-to-integer/eastern_to_integer.go

```diff
@@ -0,0 +1,26 @@
+/*
+This algorithm will convert an Eastern string number to an integer
+https://en.wikipedia.org/wiki/Eastern_Arabic_numerals
+Function receives a string as an Eastern number and outputs an integer
+*/
+
+package easterntointeger
+
+var persianNumbers = map[string]int{"۰": 0, "۱": 1, "۲": 2, "۳": 3, "۴": 4, "۵": 5, "۶": 6, "۷": 7, "۸": 8, "۹": 9}
+var arabicNumbers = map[string]int{"٠": 0, "١": 1, "٢": 2, "٣": 3, "٤": 4, "٥": 5, "٦": 6, "٧": 7, "٨": 8, "٩": 9}
+
+
+func EasternToInteger(stringNumber string) int {
```

Comment:
I didn't include errors so that the code seems as simple as possible since people usually use these repos to learn and understand algorithms rather than using the code in their projects so just returning an empty result seemed a better idea. But I can add errors if you think that it's not the case.

Reply:
People do learn from this repository, including me, but I think people should also learn the industry best practices to implement things that are production ready. Don't you agree?

Reply:
That's a valid point too and I agree. I'll add some validations and errors.